darwin/future/meta/objects/base.py

```diff
@@ -1,20 +1,22 @@
 from __future__ import annotations
 
-from typing import Generic, List, Optional, TypeVar
+from typing import Dict, Generic, List, Optional, TypeVar
 
 from darwin.future.core.client import Client
 from darwin.future.pydantic_base import DefaultDarwin
 
 R = TypeVar("R", bound=DefaultDarwin)
+Param = Dict[str, object]
 
 
 class MetaBase(Generic[R]):
     _item: Optional[R]
     client: Client
 
-    def __init__(self, client: Client, item: Optional[R] = None) -> None:
+    def __init__(self, client: Client, item: Optional[R] = None, meta_params: Optional[Param] = None) -> None:
         self.client = client
         self._item = item
+        self.meta_params = meta_params or {}
```

Comment:
There's some argument for this being `or dict()` just because `{}` is also the syntax for set delimitation, but now I'm really scraping the barrel I guess.